src/platforms/android/include.gradle

```diff
@@ -1,11 +1,8 @@
 android {
-	productFlavors {
```

Comment:
productFlavors are needed for backwards compatibility with {N} 3.x

Reply:
Yes of course, this was an auto change by the tns run command